modules/osd-create-cluster-ccs.adoc

```diff
@@ -279,6 +279,13 @@ ifdef::osd-on-gcp[]
 If you are installing a cluster into a Shared VPC, the VPC name and subnets are shared from the host project.
 ====
 endif::osd-on-gcp[]
+ifdef::osd-on-aws[]
+.. Optional: Expand *Additional security groups* and select additional custom security groups to apply to nodes in the machine pools created by default. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.
++
+By default, the security groups you specify will be added for all node types. Uncheck the *Apply the same security groups to all node types* checkbox to apply different security groups for each node type.
```

Comment:
```suggestion
By default, the security groups you specify are added for all node types. Clear the *Apply the same security groups to all node types* checkbox to apply different security groups for each node type.
```
- Avoid future tense where possible. IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense
- Use "clear" instead of "uncheck" for checkboxes. IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#checkbox

Reply:
here also: modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc - changed here also